theatre/studio/src/panels/SequenceEditorPanel/RightOverlay/PlayheadPositionPopover.tsx

```diff
@@ -76,7 +76,7 @@ const PlayheadPositionPopover: React.FC<{
       <Container>
         <Label>Playhead position</Label>
         <BasicNumberInput
-          value={sequence.position}
+          value={Number(sequence.position.toFixed(3))}
```

Comment:
The popover used to show values like `1.238947298374`. Now it shows `1.123`.

However, this commit has been erroneously rebased and the trigger for opening the popover is removed so the popover doesn't show up. Fixed in a later commit.